sorc/chgres_cube.fd/atmosphere.F90

```diff
@@ -1,29 +1,29 @@
- module atmosphere
+!> @file
+!! @brief Process atmospheric fields
+!!
+!! @author gayno NCEP/EMC
+!!
+!! Abstract: Process atmospheric fields:  Horizontally interpolate input
```

Comment:
Take out the "Abstract:" everywhere. Doxygen output already indicates what each section of text is, so we don't need to.

Reply:
Done.